### ISSUE TITLE
Error running rule disallowAtEachLeafNode

### ISSUE THREAD

**Issue by amkirwin** [minichate/jscs-ember-deprecations#1]
```
Error running rule disallowAtEachLeafNode: This is an issue with JSCS and not your codebase.
Please file an issue (with the stack trace below) at: https://github.com/jscs-dev/node-jscs/issues/new
TypeError: Cannot read property 'name' of undefined
    at node_modules/jscs-ember-deprecations/lib/rules/disallow-ateachleafnode.js:39:56
    at Array.forEach (native)
    at Object. (node_modules/jscs-ember-deprecations/lib/rules/disallow-ateachleafnode.js:37:20)
    at Array.forEach (native)
    at Object.JsFile.iterateNodesByType (atom/packages/linter-jscs/node_modules/jscs/lib/js-file.js:595:42)
    at [object Object].module.exports.check (node_modules/jscs-ember-deprecations/lib/rules/disallow-ateachleafnode.js:36:8)
    at [object Object]. (atom/packages/linter-jscs/node_modules/jscs/lib/string-checker.js:165:22)
    at Array.forEach (native)
    at [object Object].StringChecker._checkJsFile (atom/packages/linter-jscs/node_modules/jscs/lib/string-checker.js:161:31)
    at [object Object].StringChecker.checkString (atom/packages/linter-jscs/node_modules/jscs/lib/string-checker.js:106:14)at line 1 col 1
```

**Comment by minichate** [comment]
@amkirwin what version of the library are you using? 0.0.14 should have a fix for that issue.

If not, do you have a sample test case that reproduces the issue?

**Comment by amkirwin** [comment]
0.0.14 ;)

**Comment by minichate** [comment]
Can you share a snippet of code that triggers the issue? I'll try and get a fix out quickly

**Comment by amkirwin** [comment]
Yep, I don't think it's anything out of the ordinary though
```
export function fooFroup(key, groupName) {
  return computed('foo.@each', function() {
    // ...
  });
}
```

**Comment by minichate** [comment]
@amkirwin I _think_ it should be fixed in 0.0.15 now which I just published to npm! Let me know if it works for you.

**Comment by amkirwin** [comment]
:+1: 
sorted, thanks!
